server/src/endpoint/sms/message.ts

```diff
@@ -30,23 +31,6 @@ export const generateValidationCodes = (number: string) => {
   });
 };
```

Comment:
Might as well move the generateValidationCodes function to that new file as well since we will need that if we are using the same validation process.